lib/nmatrix/nmatrix.rb

```diff
@@ -534,6 +534,16 @@ def row(row_number, get_by = :copy)
     rank(0, row_number, get_by)
   end
 
+  #
+  # call-seq:
+  #     last -> Element of self.dtype
+  #
+  # Returns the last element stored in an NMatrix
+  #
+  def last
+    self[*Array.new(self.dim, -1)]
```

Comment:
This is extremely inefficient. Before making another attempt, let's please discuss this on the issue page (#481).